apps/system/build/build.js

```diff
@@ -101,18 +101,23 @@ SystemAppBuilder.prototype.integrateLockScreenInputpad = function(options) {
 SystemAppBuilder.prototype.inlineDeviceType = function(options) {
   var stagePath = options.STAGE_APP_DIR;
   var deviceType = options.GAIA_DEVICE_TYPE;
-  var featureDetectorPath = [stagePath, 'js', 'feature_detector.js'];
-  var featureDetectorFile = utils.getFile.apply(utils, featureDetectorPath);
-  var featureDetectorContent = utils.getFileContent(featureDetectorFile);
-
-  // `this.deviceType = '_GAIA_DEVICE_TYPE_';` will be replaced by real device
-  // type, take phone for example, result will be: this.deviceType = 'phone';
-  utils.writeContent(
-    featureDetectorFile,
-    featureDetectorContent.replace(
-      /this\.deviceType = \'_GAIA_DEVICE_TYPE_\';/,
-      'this.deviceType = \'' + deviceType + '\';')
-  );
+
+  utils.ls(utils.getFile(stagePath, 'js'), true).forEach(function(file) {
```

Comment:
I would suggest to revert this part. We should let as less people as possible to know we have this variable because it is not standard. They should use `Service.query('getDeviceType');` to know device type.